xml/art_jeos_quickstart.xml

```diff
@@ -216,6 +216,35 @@
     </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>systemd-coredump disabled</term>
+     <listitem>
+      <para>
+       Because of the small amount of memory and disk space available by default on JeOS images,
+       systemd-coredump is disabled by default on JeOS images.
+       If your usage scenario requires to collect kernel core dumps for analyzing application
+       crashes:
+       <itemizedlist>
+        <listitem>
+         <para>
+	  Install the <package>systemd-coredump</packages> package,
+         </para>
+        </listitem>
+        <listitem>
+         <para>
```

Comment:
Not true, either do a reboot or run `sysctl --system` to apply it immediately.

Reply:
Has this comment been addressed @vmoutoussamy ?